index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microvers</title>
+</head>
```

Comment:
Kindly link your CSS file to your HTML file. You can do it with this code piece `<link rel="stylesheet" href="styles.css">` So you will have your fancy color for the h1 element.